Analyzer/test/MyAnalysis.C

```diff
@@ -159,7 +177,7 @@ int main(int argc, char *argv[])
     if(runOnCondor)
     {
         char thistFile[128];
-        sprintf(thistFile, "MyAnalysis_%s_%d.root", dataSets.c_str(), startFile);
+        sprintf(thistFile, "MyHadTrigAnalysis_%s_%d.root", dataSets.c_str(), startFile);
         histFile = thistFile;
```

Comment:
Why did you change the name of the output file name? In general it is fine but it might break some down stream code that looks for that specific name.